src/components/layout/areas/MlEphantConversationPaneWrapper.tsx

```diff
@@ -36,7 +40,7 @@ export function MlEphantConversationPaneWrapper(props: AreaTypeComponentProps) {
           <LayoutPanelHeader
             id={props.layout.id}
             icon="sparkles"
-            title="Text-to-CAD"
+            title="Zookeeper"
```

Comment:
Noticed that the tootip isn't updated 

<img width="234" height="61" alt="Image" src="https://github.com/user-attachments/assets/48122068-e517-4501-a77d-9b03c2ebebac" />